src/data/stig_multipliers.ts

```diff
@@ -1428,6 +1460,52 @@ const DPS_STIG_MULTIPLIERS = Object.freeze({
       }),
     ],
   },
+  "Rebel Blades (Physical)": {
+    "Mei T": [
+      new Multiplier({
+        active: true,
+        name: "Mei T",
+        type: Type.PhysDealt,
+        value: 13,
+      }),
+      new Multiplier({
+        active: true,
+        name: "Mei T: Gain 4% phys DMG per hit. Max 5 stacks.",
+        type: Type.PhysDealt,
+        value: 20,
+      }),
+    ],
+    "Fu Hua M": [
+      new Multiplier({
+        active: true,
+        name: "Fu Hua M",
+        type: Type.CritDmg,
+        value: 15,
+      }),
+    ],
+    "Fu Hua B": [
+      new Multiplier({
+        active: true,
+        name: "Fu Hua B",
+        type: Type.CritRate,
+        value: 12,
+      }),
+      new Multiplier({
+        active: false,
```

Comment:
I think it's fine to have this one set to active by default

Reply:
Incorporated in latest commit